### ISSUE TITLE
Getting a ClientAuthError while running Inferno tests for Standalone Patient App

### ISSUE THREAD

**Issue by nimesh13** [Azure-Samples/azure-health-data-and-ai-samples#114]
I have followed all the steps in the [deployment document](https://github.com/Azure-Samples/azure-health-data-and-ai-samples/blob/main/samples/patientandpopulationservices-smartonfhir-oncg10/docs/deployment.md) for implementing SMART On FHIR

While running the Inferno tests for Standalone Patient App, the redirect URL throws out this error:
<img width="1512" alt="image" src="https://github.com/Azure-Samples/azure-health-data-and-ai-samples/assets/26820845/e9779dc8-5d98-4942-b558-eba255fc3c17">

<img width="1512" alt="image" src="https://github.com/Azure-Samples/azure-health-data-and-ai-samples/assets/26820845/a3da3dc2-a0f8-4f3e-9f4d-79fa1c0796a2">

The error on the screen leads me to this error:
{"error":"invalid_tenant","error_description":"AADSTS90002: Tenant 'undefined' not found. Check to make sure you have the correct tenant ID and are signing into the correct cloud. Check with your subscription administrator, this may happen if there are no active subscriptions for the tenant.\r\nTrace ID: xyz\r\nCorrelation ID: xyz\r\nTimestamp: 2023-08-25 15:11:12Z","error_codes":[90002],"timestamp":"2023-08-25 15:11:12Z","trace_id":"xyz","correlation_id":"xyz","error_uri":"https://login.microsoftonline.com/error?code=90002"}

^Masking the IDs above.

[Commented on](https://github.com/Azure-Samples/azure-health-data-and-ai-samples/issues/111#issuecomment-1673473624) a different but similar issue. 

Hoping for a quick resolution since these issues have been in Open for quite some time.

**Comment by EXPEkesheth** [comment]
@nimesh13 - can you please confirm the same tenantid where SMART on FHIR deployment script is installed is used?

**Comment by nimesh13** [comment]
@EXPEkesheth yes the same tenantid is used as the azd env variable where the script is installed.

<img width="448" alt="image" src="https://github.com/Azure-Samples/azure-health-data-and-ai-samples/assets/26820845/8600012d-b4e3-4684-8724-7a2d887b5a94">

<img width="476" alt="image" src="https://github.com/Azure-Samples/azure-health-data-and-ai-samples/assets/26820845/bde96514-929b-4836-8e33-819512b6ea59">

**Comment by EXPEkesheth** [comment]
Thanks @nimesh13 - we will try to repro the issue and report back on the findings.

**Comment by EXPEkesheth** [comment]
@nimesh13 - Can you please trying executing 'azd up' command? Please let us know what you identify after running the command or it helps resolve the issue.

**Comment by nimesh13** [comment]
@EXPEkesheth - thanks for replying. I did perform `azd up` and this error no longer appears. I am able to login and grant access and continue with the rest of the inferno test suite. But I have a question - does performing `azd up` automatically pull the latest code from the repository? I haven't updated my local repository with the remote ever since I have raised this issue. The reason I am asking this is because I am facing some new issues in inferno which I may have seen in other github issues. So I just want to make sure what the right process is.
